tests/tpch/conftest.py

```diff
@@ -143,6 +145,24 @@ def cluster_spec(scale):
         idle_timeout="1h",
         wait_for_workers=True,
         scheduler_vm_types=["m6i.xlarge"],
+        backend_options={
+            "ingress": [
+                {
+                    "ports": [
+                        443,
+                        8786,
+                        15003,
+                        22,
+                        7077,
+                        8787,
+                        4040,
+                        8080,
+                        15002,
```

Comment:
It would be very helpful for future work to document what these ports are for (once you're done)

Reply:
Ya, agreed. I'm kinda hoping platform can adjust ports to add 4040 to default spark ports and this can all go away. Had to specify all of them here since I'm overriding the backend options in order to get 4040 in. 

@ntabris What do you think?

Reply:
I'd want to understand what's on 4040, if we should/can put auth in front of it, etc.

FYI you can use `get_spark_cluster(_open_debug_ports=True)` instead of what you're doing here.

Reply:
> at the point of the code here the cluster is already made

No idea what you mean, you can only set ports to open when making a new cluster.